### DISCUSSION TITLE
Please add gists management to Android app

### DISCUSSION THREAD

**Discussion by dmusicant-dk** [community/community#3229] [Mobile] [answered]
The current GitHub app does not show your gists.

**Comment by amrehman** [comment, marked as answer]
Thanks a lot for your feedback!

I've added your note to our internal issue where we are tracking adding gists. If you have any other feedback or feature requests, please keep them coming!

**Reply by kanition** [reply]
Still nothing in 2025

**Reply by mohamedxelturkey-cpu** [reply]
U
